Assets/UI/Scripts/NewGameUI.gd

```diff
@@ -0,0 +1,44 @@
+extends Control
+
+var parent = null
+
+# Deactivate everything irrelevant for the time being
+func _ready() -> void:
+	var nodes = {
+		"Scenario": null,
+		"RandomMap": null,
+		"ResourceDensitySlider": null,
+		"Traders": null,
+		"Disasters": null
+	}
+
+	nodes = find_nodes(get_tree().get_root(), nodes)
```

Comment:
...since right now this could just be `find_nodes(get_tree().get_root(), nodes)`.